verification/cbmc/proofs/cbmc_negative_tests/cbmc_negative_tests_harness.c

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/**
+ * A basic negative assertion should fail
+ * if CBMC was run at all.
+ */
+void assert_check() {
+    assert(1 == 0);
+}
+
+/**
+ * A negative test for --bounds-check flag
+ */
+void bounds_check() {
+    char test[10];
+    test[12] = 'x';
+}
+
+/**
+ * A negative test for --conversion-check flag
+ */
+void conversion_check() {
+    unsigned long src;
+    unsigned dst = src;
```

Comment:
Whether this fails depends on the architecture.  It might be more reliable to say `uint32_t` and `uint64_t`